terraform/cdn.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_cloudfront_distribution" "stage" {
```

Comment:
I'm doing this same type of cloudfront as cdn stuff for www.firefox.com here https://github.com/mozmeao/infra-services/tree/master/aws/global/modules/cloudfront Do you think there's any value to extracting a cloudfront module to share? (I'd lean towards no, because there's not a ton of repetition, almost all the code here is a choice that can vary).

Reply:
Hi @duallain thanks for looking at the PR!
I really think there's value in creating a TF module, I didn't do it because I thought no one else would be creating a CF distribution, and our middle term plan is to move to GCP.

We could safe ourselves a bunch of code having the CF setup + Domain registration + ACM certs abstracted into a module.
Do you want to draft something at github.com/mozilla-it/terraform-modules/aws?